packages/code-du-travail-frontend/src/outils/DureePreavisDemission/steps/situation.js

```diff
@@ -168,3 +82,14 @@ export function recapSituation(criteria) {
       null
     );
 }
+
+export function getSituationsFor(obj) {
```

Comment:
can you make this function generic getSituationFor(data, values) => situations[]

Reply:
OK

Reply:
Done